### ISSUE TITLE
Login failed

### ISSUE THREAD

**Issue by chylex** [Daskiworks/ghwatch#109]
I tried both my randomly generated password copied from a password manager, and a personal access token, both keep reporting "GitHub login failed, username or password is invalid". Right now the app simply doesn't work for me, don't know what's the problem.

**Comment by velias** [comment]
Hi. Are you able to login on github.com website with given random password? If yes, does it contain some special characters - anything else that common letters or numbers?

**Comment by chylex** [comment]
Yes, I can login with a browser on my phone, and yes there are multiple utf-8 characters in the password.

**Comment by velias** [comment]
Thanks, I'll check app code for the pwd encoding into auth request

**Comment by kkm000** [comment]
@chylex, were you able to log in before, or was that the very first time you tried to log on with the app? I am asking because another issue #108 was opened 2 days before yours, and I also having that problem (but not this logon issue, I am logging on just fine).

I am just another user of the app. Just wondering if something could have possibly changed on GitHub's side around that time. May be a coincidence, may be not, just trying to connect the dots.

**Comment by chylex** [comment]
First time.

**Comment by kkm000** [comment]
Ah, not likely related to #108 then!

First time.
